examples/create-transaction.ts

```diff
@@ -18,5 +18,5 @@ const newTransaction = await client.SendToRecipients(
 );
 console.log('SendToRecipients response:', newTransaction);
 
-const tx = await client.GetTransaction(newTransaction.id);
+const tx = await client.GetTransaction(newTransaction.id() as string);
```

Comment:
I don't like this `as string` part, but I'm aware that it depends on `ts-sdk` implementation. 
Maybe we could recommend creators to describe function oveloads to make this more typescript-friendly?

```typescript
// Describe the function overloads
id(): number[];
id(enc: 'hex'): string;

// Implement the function using these overloads
id(enc?: 'hex'): number[] | string {
    const id = [...this.hash()];
    id.reverse();
    if (enc === 'hex') {
      return toHex(id);
    }
    return id;
  }
```

Reply:
Yes, I agree! I prefer overloads instead of this. 
Would you like to add a suggestion in this library or would you like me to do that?

Reply:
Would be great if you could 😄

Reply:
https://github.com/bitcoin-sv/ts-sdk/issues/52